<!DOCTYPE html>
<html lang='en'>
<head>
<title>0004-don-t-use-usr-bin-env-in-shebang.patch - rpms/groff.git - groff</title>
<meta name='generator' content='cgit v1.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://src.fedoraproject.org/cgit/rpms/groff.git/atom/0004-don-t-use-usr-bin-env-in-shebang.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='https://src.fedoraproject.org/git/rpms/groff.git' title='rpms/groff.git Git repository'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 5 && tokens[4] == 'rpms' && tokens[5] != '') {
    repo = tokens[5].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='rpms/groff.git' href='/cgit/rpms/groff.git/'>rpms/groff.git</a></td><td class='form'><form method='get'>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f23'>f23</option>
<option value='f24'>f24</option>
<option value='f25'>f25</option>
<option value='f26'>f26</option>
<option value='f27'>f27</option>
<option value='f28'>f28</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' value='switch'/></form></td></tr>
<tr><td class='sub'>groff</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/rpms/groff.git/'>summary</a><a href='/cgit/rpms/groff.git/refs/'>refs</a><a href='/cgit/rpms/groff.git/log/0004-don-t-use-usr-bin-env-in-shebang.patch'>log</a><a class='active' href='/cgit/rpms/groff.git/tree/0004-don-t-use-usr-bin-env-in-shebang.patch'>tree</a><a href='/cgit/rpms/groff.git/commit/0004-don-t-use-usr-bin-env-in-shebang.patch'>commit</a><a href='/cgit/rpms/groff.git/diff/0004-don-t-use-usr-bin-env-in-shebang.patch'>diff</a><a href='/cgit/rpms/groff.git/stats/0004-don-t-use-usr-bin-env-in-shebang.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/rpms/groff.git/log/0004-don-t-use-usr-bin-env-in-shebang.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/rpms/groff.git/tree/'>root</a>/<a href='/cgit/rpms/groff.git/tree/0004-don-t-use-usr-bin-env-in-shebang.patch'>0004-don-t-use-usr-bin-env-in-shebang.patch</a></div><div class='content'>blob: 5c8e2e758a4aa6891be1ac634896b1e35e2eced0 (<a href='/cgit/rpms/groff.git/plain/0004-don-t-use-usr-bin-env-in-shebang.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
<a id='n44' href='#n44'>44</a>
<a id='n45' href='#n45'>45</a>
<a id='n46' href='#n46'>46</a>
<a id='n47' href='#n47'>47</a>
<a id='n48' href='#n48'>48</a>
<a id='n49' href='#n49'>49</a>
<a id='n50' href='#n50'>50</a>
<a id='n51' href='#n51'>51</a>
<a id='n52' href='#n52'>52</a>
<a id='n53' href='#n53'>53</a>
<a id='n54' href='#n54'>54</a>
<a id='n55' href='#n55'>55</a>
<a id='n56' href='#n56'>56</a>
<a id='n57' href='#n57'>57</a>
<a id='n58' href='#n58'>58</a>
<a id='n59' href='#n59'>59</a>
</pre></td>
<td class='lines'><pre><code>From e263e19aa1c63dbcbe710e8aae79c8e298606e4c Mon Sep 17 00:00:00 2001
From: Peter Schiffer &lt;pschiffe&#64;redhat.com&gt;
Date: Tue, 4 Nov 2014 14:49:57 +0100
Subject: [PATCH] don't use /usr/bin/env in shebang

There might be an issue that the script is executed with unwanted version of
<span class="hl kwa">&lt;lang&gt; if that language is provided by enabled dynamic software collection.</span>

Resolves: #987069
<span class="hl kwb">---</span>
 contrib/chem/chem.pl       | 2 +-
 contrib/groffer/groffer.pl | 2 +-
 contrib/groffer/roff2.pl   | 2 +-
 src/roff/grog/grog.pl      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/contrib/chem/chem.pl b/contrib/chem/chem.pl
index 1a8b3cc..15eda5f 100755
<span class="hl kwb">--- a/contrib/chem/chem.pl</span>
<span class="hl kwa">+++ b/contrib/chem/chem.pl</span>
<span class="hl kwd">&#64;&#64; -1,4 +1,4 &#64;&#64;</span>
<span class="hl kwb">-#! /usr/bin/env perl</span>
<span class="hl kwa">+#! /usr/bin/perl</span>
 
 # chem - a groff preprocessor for producing chemical structure diagrams
 
diff --git a/contrib/groffer/groffer.pl b/contrib/groffer/groffer.pl
index 4e2e575..913e8eb 100755
<span class="hl kwb">--- a/contrib/groffer/groffer.pl</span>
<span class="hl kwa">+++ b/contrib/groffer/groffer.pl</span>
<span class="hl kwd">&#64;&#64; -1,4 +1,4 &#64;&#64;</span>
<span class="hl kwb">-#! /usr/bin/env perl</span>
<span class="hl kwa">+#! /usr/bin/perl</span>
 
 # groffer - display groff files
 
diff --git a/contrib/groffer/roff2.pl b/contrib/groffer/roff2.pl
index f0ca9f2..1b8577f 100755
<span class="hl kwb">--- a/contrib/groffer/roff2.pl</span>
<span class="hl kwa">+++ b/contrib/groffer/roff2.pl</span>
<span class="hl kwd">&#64;&#64; -1,4 +1,4 &#64;&#64;</span>
<span class="hl kwb">-#! /usr/bin/env perl</span>
<span class="hl kwa">+#! /usr/bin/perl</span>
 
 # roff2* - transform roff files into other formats
 
diff --git a/src/roff/grog/grog.pl b/src/roff/grog/grog.pl
index fb7b54c..9baa869 100644
<span class="hl kwb">--- a/src/roff/grog/grog.pl</span>
<span class="hl kwa">+++ b/src/roff/grog/grog.pl</span>
<span class="hl kwd">&#64;&#64; -1,4 +1,4 &#64;&#64;</span>
<span class="hl kwb">-#! /usr/bin/env perl</span>
<span class="hl kwa">+#! /usr/bin/perl</span>
 # grog - guess options for groff command
 # Inspired by doctype script in Kernighan &amp; Pike, Unix Programming
 # Environment, pp 306-8.
<span class="hl kwb">-- </span>
1.9.3

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='https://git.zx2c4.com/cgit/about/'>cgit v1.1</a> at 2018-03-03 16:33:45 +0000</div>
</div> <!-- id=cgit -->
</body>
</html>
